docs/Database_Schema.md

```diff
@@ -448,3 +448,22 @@ Create TABLE IF NOT EXISTS "marketcap" (
     PRIMARY KEY ("last_updated_at")
 );
 ```
+
+## 16. Table **tx_pool**
+
+### _Schema_
+
+| Column            | Data Type | PK | Not NULL | Default  |Description|
+|:----------------- |:--------- |:--:|:--------:| -------- | --------- |
+| hash              | TEXT      | Y  | Y        |          | The hash of transaction|
+| tx                | BLOB      | Y  | Y        |          | The transaction serialized to binary |
+
+### _Create Script_
+
+```sql
+CREATE TABLE IF NOT EXISTS tx_pool (
+    hash   TEXT    NOT NULL,
+    tx     BLOB    NOT NULL,
```

Comment:
```suggestion
    tx_hash_text   TEXT    NOT NULL,
    tx_bytes     BLOB    NOT NULL,
```

Reply:
It's not too long. I don't think it's necessary to describe the form.

Reply:
I'd like to prohibit the use of this table by joining other tables.

Reply:
Then it can replace it with an explanation.